contracts/integrations/test/framework/assertions/moveStake.ts

```diff
@@ -86,61 +86,50 @@ export function validMoveStakeAssertion(
+            const updatedPools = updateNextEpochBalances(globalStake, ownerStake, pools, from, to, amount);
 
-                // Update local balances to match the expected result of this `moveStake` operation
-                const updatedPools = updateNextEpochBalances(globalStake, ownerStake, pools, from, to, amount);
+            // Fetches on-chain owner stake balances and checks against local balances
+            const ownerUndelegatedStake = {
+                ...new StoredBalance(),
+                ...(await stakingWrapper.getOwnerStakeByStatus(owner, StakeStatus.Undelegated).callAsync()),
+            };
+            const ownerDelegatedStake = {
+                ...new StoredBalance(),
+                ...(await stakingWrapper.getOwnerStakeByStatus(owner, StakeStatus.Delegated).callAsync()),
+            };
+            expect(ownerUndelegatedStake).to.deep.equal(ownerStake[StakeStatus.Undelegated]);
+            expect(ownerDelegatedStake).to.deep.equal(ownerStake[StakeStatus.Delegated].total);
 
-                // Fetches on-chain owner stake balances and checks against local balances
-                const ownerUndelegatedStake = {
-                    ...new StoredBalance(),
-                    ...(await stakingWrapper.getOwnerStakeByStatus(owner, StakeStatus.Undelegated).callAsync()),
-                };
-                const ownerDelegatedStake = {
-                    ...new StoredBalance(),
-                    ...(await stakingWrapper.getOwnerStakeByStatus(owner, StakeStatus.Delegated).callAsync()),
-                };
-                expect(ownerUndelegatedStake).to.deep.equal(ownerStake[StakeStatus.Undelegated]);
-                expect(ownerDelegatedStake).to.deep.equal(ownerStake[StakeStatus.Delegated].total);
+            // Fetches on-chain global stake balances and checks against local balances
+            const globalUndelegatedStake = await stakingWrapper
+                .getGlobalStakeByStatus(StakeStatus.Undelegated)
+                .callAsync();
+            const globalDelegatedStake = await stakingWrapper.getGlobalStakeByStatus(StakeStatus.Delegated).callAsync();
+            expect(globalUndelegatedStake).to.deep.equal(globalStake[StakeStatus.Undelegated]);
+            expect(globalDelegatedStake).to.deep.equal(globalStake[StakeStatus.Delegated]);
 
-                // Fetches on-chain global stake balances and checks against local balances
-                const globalUndelegatedStake = await stakingWrapper
-                    .getGlobalStakeByStatus(StakeStatus.Undelegated)
+            // Fetches on-chain pool stake balances and checks against local balances
+            for (const poolId of updatedPools) {
+                const stakeDelegatedByOwner = await stakingWrapper
+                    .getStakeDelegatedToPoolByOwner(owner, poolId)
                     .callAsync();
-                const globalDelegatedStake = await stakingWrapper
```

Comment:
How come we removed this check?

Reply:
it's there, just got moved up a few lines 🙂